plugins/internal/tengoutil/secure_script_test.go

```diff
@@ -55,4 +55,15 @@ func TestNewSecureScript(t *testing.T) {
 		_, err = s.Compile()
 		assert.NoError(t, err)
 	})
+
+	t.Run("Allows import of custom http module", func(t *testing.T) {
+		s, err := NewSecureScript(([]byte)(heredoc.Doc(`
+			http := import("http")
+			response := http.get("http://example.com")
```

Comment:
should we align with this?
https://github.com/goto/meteor/blob/main/plugins/extractors/http/README.md#execute_requestrequests-response